runner/cli.ts

```diff
@@ -57,26 +58,26 @@ function run(env, args, output, callback) {
 // Note that we're cheating horribly here. Ideally all of this logic is within
 // wct-sauce. The trouble is that we also want WCT's configuration lookup logic,
 // and that's not (yet) cleanly exposed.
-function runSauceTunnel(env, args, output, callback) {
-  var done = wrapCallback(output, callback);
+export function runSauceTunnel(env: void, args: string[], output: NodeJS.WritableStream, callback: (err: any) => void) {
+  const done = wrapCallback(output, callback);
 
-  var diskOptions = config.fromDisk();
-  var baseOptions = diskOptions.plugins && diskOptions.plugins.sauce || diskOptions.sauce || {};
+  const diskOptions = config.fromDisk();
+  const baseOptions: config.Config = diskOptions.plugins && diskOptions.plugins['sauce'] || diskOptions.sauce || {};
 
   Plugin.get('sauce', function(error, plugin) {
     if (error) return done(error);
 
-    var parser = require('nomnom');
+    const parser = require('nomnom');
     parser.script('wct-st');
     parser.options(_.omit(plugin.cliConfig, 'browsers', 'tunnelId'));
-    var options = _.merge(baseOptions, parser.parse(args));
+    const options = _.merge(baseOptions, parser.parse(args));
 
-    var wctSauce = require('wct-sauce');
+    const wctSauce = require('wct-sauce');
     wctSauce.expandOptions(options);
 
-    var emitter = new events.EventEmitter();
-    new CliReporter(emitter, output, {});
-    wctSauce.startTunnel(options, emitter, function(error, tunnelId) {
+    const emitter = new events.EventEmitter();
+    new CliReporter(emitter, output, <config.Config>{});
+    wctSauce.startTunnel(options, emitter, function(error: any, tunnelId: string) {
```

Comment:
Not in a node callback, the only contract is that the operation was successful if `error` is `null`. AFAIK `error` can properly be anything.